docs/config/coordinator.md

```diff
@@ -26,6 +26,28 @@ The URL to the load-test API server that will run the agent.
 
 The maximum number of concurrently active users to be run across the whole load-agent cluster.
 
+### BrowserAgents
+
+*[]cluster.LoadAgentConfig*
+
+#### Id
+
+*string*
+
+The unique identifier of the browser load-agent to be created and run.
+
+#### ApiURL
+
+*string*
+
+The URL to the load-test API server that will run the agent. Since this is an agent as well, it will be the same as the load-test API server. This should not be confused with the LTBrowser API server which is a separate server that is controlled by the load-test API server.
+
+### MaxActiveBrowserUsers
+
+*int*
+
+The maximum number of concurrently active browser users to be run across the entire load-agent cluster. Note that there is also a MaxActiveBrowserUsers setting in config/config.json which limits the number of browser users per individual agent, while this setting limits the total number of browser users across all agents in the cluster. The MaxActiveBrowserUsers value in the cluster configuration should be less than the product of MaxActiveBrowserUsers per browser agent multiplied by the number of browser agents.
```

Comment:
We should add a note here stating that this cannot be larger than MaxActiveUsers, since MaxActiveUsers is the cap of total users connected.